UnicodeJsps/src/test/java/org/unicode/jsptest/TestMultivalued.java

```diff
@@ -0,0 +1,53 @@
+package org.unicode.jsptest;
+
+import com.ibm.icu.text.UnicodeSet;
+import org.junit.jupiter.api.Test;
+import org.unicode.jsp.UnicodeSetUtilities;
+import org.unicode.unittest.TestFmwkMinusMinus;
+
+public class TestMultivalued extends TestFmwkMinusMinus {
+    @Test
+    public void TestScx1Script() {
+        // As of 2023-11-24, scx was not working properly
```

Comment:
Please remove this comment. Putting it in together with the fix is confusing for future readers. The test code here shows that it works.

Reply:
ok